lib/install.js

```diff
@@ -273,13 +268,16 @@ const fs = require('fs'),
             async.waterfall([
                 function(next) {
                     options.nReplies = undefined; // -i
-                    options.session = new novacom.Session(options.device, next);
+                    if(!options.session){
+                        options.session = new novacom.Session(options.device, next);
+                    } else {
+                        next(null, options.session);
+                    }
                 },
                 function(session, next) {
-                    options.session = session;
                     if (options.opkg) {
                         // FIXME: Need more consideration whether this condition is necessary or not.
-                        if (options.session.getDevice().username !== 'root') {
+                        if (session.getDevice().username !== 'root') {
```

Comment:
#272, options.session = new novacom.Session(options.device, next); 
already options.session is assigned.
I prefer 
if (options.session.getDevice().username !== 'root') {

Reply:
No problem to use "options.session.getDevice". 
However, function(session) 's session value becomes "unused value".

Reply:
Change "options.session"

Reply:
Changed.